dephell/config/builders.py

```diff
@@ -14,7 +14,7 @@ def build_config(parser):
     config_group = parser.add_argument_group('Configuration file')
     config_group.add_argument('-c', '--config', help='path to config file.')
     config_group.add_argument('-e', '--env', default='main', help='environment in config.')
-
+    config_group.add_argument('-o', '--owner', help='name of owner.')
```

Comment:
1. Please, remove short flag `-o` 
1. Place it in `build_other` instead of `build_config`.

Reply:
Moved to `build_other` and also added info to docs/param.md